templates/kurl-proxy-deployment.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if .Values.kurlProxy.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kurl-proxy-kotsadm
+  labels:
+    app: kurl-proxy-kotsadm
+    {{- include "admin-console.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: kurl-proxy-kotsadm
+  template:
+    metadata:
+      labels:
+        app: kurl-proxy-kotsadm
+        {{- include "admin-console.labels" . | nindent 8 }}
+    spec:
+      {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: proxy
+        image: {{ .Values.images.kurlProxy }}
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: NODE_PORT
+          value: "8800"
```

Comment:
does this need to be `{{ .Values.kurlProxy.nodePort }}`?

Reply:
Yes it does (if we don't want to make `8800` the nodeport, of course)

Reply:
Unsetting...